src/mark.js

```diff
@@ -145,21 +147,36 @@ class Mark {
      */
     createDiacriticsRegExp(str) {
         const dct = [
-            "aÀÁÂÃÄÅàáâãäåĀāąĄ",
-            "cÇçćĆčČ",
-            "dđĐďĎ",
-            "eÈÉÊËèéêëěĚĒēęĘ",
-            "iÌÍÎÏìíîïĪī",
-            "lłŁ",
-            "nÑñňŇńŃ",
-            "oÒÓÔÕÕÖØòóôõöøŌō",
-            "rřŘ",
-            "sŠšśŚ",
-            "tťŤ",
-            "uÙÚÛÜùúûüůŮŪū",
-            "yŸÿýÝ",
-            "zŽžżŻźŹ"
-        ];
+            "aàáâãäåāą",
```

Comment:
Writing the array this way – with a separate line for each property – is clean if it's not so large. 
In this case please try to fill out the line length of 80chars for the `const dct` array in order to reduce the function height